test/integration/install/test_install.py

```diff
@@ -44,6 +85,99 @@ def test_install_tt(tt_cmd, tmpdir):
     assert re.search(r"Tarantool CLI version \d+.\d+.\d+", start_output)
 
 
+@pytest.mark.slow
+def test_install_uninstall_tt_specific_commit(tt_cmd, tmpdir):
+    configPath = os.path.join(tmpdir, config_name)
+    # Create test config
+    with open(configPath, 'w') as f:
+        f.write('tt:\n  app:\n    bin_dir:\n    inc_dir:\n')
+
+    # Install specific tt's commit.
+    install_cmd = [tt_cmd, "--cfg", configPath, "install", "tt", "2df3077"]
+    instance_process = subprocess.Popen(
+        install_cmd,
+        cwd=tmpdir,
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+
+    # Check that the process shutdowned correctly.
+    instance_process_rc = instance_process.wait()
+    assert instance_process_rc == 0
+
+    installed_cmd = [tmpdir + "/bin/tt", "version"]
+    installed_program_process = subprocess.Popen(
+        installed_cmd,
+        cwd=tmpdir + "/bin",
+        stderr=subprocess.STDOUT,
+        stdout=subprocess.PIPE,
+        text=True
+    )
+    start_output = installed_program_process.stdout.readline()
+    assert re.search(r"Tarantool CLI version 1.0.1", start_output)
+    assert re.search(r"commit: 2df3077", start_output)
+
+    # Uninstall specific tt's commit.
+    uninstall_cmd = [tt_cmd, "--cfg", configPath, "uninstall", "tt=2df3077"]
```

Comment:
This should't work. We accept only `uninstall tt <version>` format. I don't know why this tests passes.

Reply:
Oh... I know why. `tt uninstall` invokes tt 1.0.1 version, which removes itself. ****